kolibri/plugins/coach/assets/src/views/index.vue

```diff
@@ -94,10 +94,11 @@
     vuex: {
       getters: {
         pageName: state => state.pageName,
-        className,
         isSuperuser: coreGetters.isSuperuser,
         isAdmin: coreGetters.isAdmin,
         isCoach: coreGetters.isCoach,
+        classList: state => state.classList,
+        classId: state => state.classId,
```

Comment:
Since the channel switcher component is currently responsible for changing the page, I'm wondering if it would make more sense to move these getters inside the switcher component?

In other words, passing in the list and current ID as _input_ props makes more sense if the switcher's _output_ was also decoupled.

Moving the other direction, an alternative might be to move the `changeClass` method _outside_ the switcher to the parent component, and listen for the change event?